spring/boot3-autoconfigure/src/main/java/com/linecorp/armeria/internal/spring/ArmeriaConfigurationUtil.java

```diff
@@ -463,5 +465,19 @@ public static long parseDataSize(String dataSizeText) {
         }
     }
 
+    /**
+     * Remove duplicate {@link Port} entries within the list.
+     * If there are duplicates, keep the preceding {@link Port}.
+     * The criteria for duplication are based on {@link Port#port}, {@link Port#address}, and {@link Port#ip}.
```

Comment:
We should link to the valid public method if you want to write the Javadoc.
`{@link Port#getPort()}, {@link Port#getAddress()}, and {@link Port#getIp()}`